### DISCUSSION TITLE
The side panel doesn't have any border or shadow when another overlay is open

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#23612] [Q&A] [answered]
This looks very weird to me. There's no way to tell where the border is.
1. Open the in-game changelog
2. (Optional) Scroll down
3. Open the notifications sidebar

Screenshots:
![Screenshot from 2023-05-21 11-37-58](https://github.com/ppy/osu/assets/101510334/2e6c354f-0534-4f1a-8a54-d5171147a179)
![Screenshot from 2023-05-21 11-45-35](https://github.com/ppy/osu/assets/101510334/dbf9c3e7-7a5f-4e1b-9994-306eb7dd0bdf)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23637
